### PR TITLE
newsletter 2019-09-06

### PR DESCRIPTION
**Pull request by ryan-shea**

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -899,13 +899,13 @@ def render_new_bounty_roundup(to_email):
...
 <p>
-Keep your excitement in check for the next round of Gitcoin Grants CLR matching. Inch by inch, we're getting closer to launch, and we can't wait to see what our community does this time. Have any questions about Grants? Check out the homepage <a href="https://gitcoin.co/grants/">here</a> and send us a tweet. We're happy to help.
+It's new feature time. Our devs have been working hard on the latest iteration of the Dashboard, which is now a your one stop shop for a high level overview of all your activity on the Gitcoin platform. Monitor your active bounties, applications, and submissions. The dashboard is now live at <a href="gitcoin.co/dashboard">https://gitcoin.co/dashboard/</a>.
 </p>
 <p>
-For those who we saw at ETHBerlin, it was great to see you! We're decompressing after a phenomenal month of travel. Tweet at us with any photos, and we'll give them a retweet! Our Twitter is @gitcoin.
+Inch by inch, row by row, it some time to make Gitcoin Grants grow. We promise: we're closer to launch, and we can't wait to see what our community does this time. Have any questions about Grants? Check out the homepage <a href="https://gitcoin.co/grants/">here</a> or send us a tweet.
```
Comment (owocki) [inline app/retail/emails.py]: "it some time" - is this correct grammer?

Comment (owocki) [reply app/retail/emails.py]: > send us a tweet  

link this?

Comment (ryan-shea) [reply app/retail/emails.py]: fixed below

**Comment by thelostone-mc** [comment]
Deployed ! closing this out

**Pull request closed without merging**